checker/src/org/checkerframework/checker/index/upperbound/OffsetEquation.java

```diff
@@ -387,7 +387,7 @@ private static int parseInt(String intLiteral) {
         if (intLiteral.isEmpty()) {
             return 0;
         }
-        return Integer.valueOf(intLiteral);
+        return Long.valueOf(intLiteral).intValue();
```

Comment:
Is there a strong reason why `OffsetEquation` only works with `int`s? Should we change the whole class to primarily operate on `long`s?